worlds/timespinner/Locations.py

```diff
@@ -18,257 +18,257 @@ def get_locations(world: Optional[MultiWorld], player: Optional[int]) -> Tuple[L
-        LocationData('Lower Lake Serene', 'T chest',  1337109),
-        LocationData('Lower Lake Serene', 'Past the eels',  1337110),
-        LocationData('Lower Lake Serene', 'Underwater pedestal',  1337111),
-        LocationData('Caves of Banishment (upper)', 'Banishment shroom jump room',  1337112, lambda state: state._timespinner_has_doublejump(world, player)),
-        LocationData('Caves of Banishment (upper)', 'Banishment secret room',  1337113),
-        LocationData('Caves of Banishment (upper)', 'Banishment bottom left',  1337114),
-        LocationData('Caves of Banishment (upper)', 'Banishment single shroom room',  1337115),
-        LocationData('Caves of Banishment (upper)', 'Banishment jackpot room chest 1',  1337116, lambda state: state._timespinner_has_forwarddash_doublejump(world, player)),
-        LocationData('Caves of Banishment (upper)', 'Banishment jackpot room chest 2',  1337117, lambda state: state._timespinner_has_forwarddash_doublejump(world, player)),
-        LocationData('Caves of Banishment (upper)', 'Banishment jackpot room chest 3',  1337118, lambda state: state._timespinner_has_forwarddash_doublejump(world, player)),
-        LocationData('Caves of Banishment (upper)', 'Banishment jackpot room chest 4',  1337119, lambda state: state._timespinner_has_forwarddash_doublejump(world, player)),
-        LocationData('Caves of Banishment (upper)', 'Banishment pedestal',  1337120),
-        LocationData('Caves of Banishment (Maw)', 'Last chance before Maw',  1337121, lambda state: state._timespinner_has_doublejump(world, player)),
-        LocationData('Caves of Banishment (Maw)', 'Plasma Crystal', 1337173, lambda state: state.has_any({'Gas Mask', 'Talaria Attachment'}, player)),
+        LocationData('Refugee Camp', 'Refugee Camp: Neliste\'s Bra',  1337086),
+        LocationData('Refugee Camp', 'Refugee Camp: Storage chest 3',  1337087),
+        LocationData('Refugee Camp', 'Refugee Camp: Storage chest 2',  1337088),
+        LocationData('Refugee Camp', 'Refugee Camp: Storage chest 1',  1337089),
+        LocationData('Forest', 'Forest: Refugee camp roof',  1337090),
+        LocationData('Forest', 'Forest: Bat jump ledge',  1337091, lambda state: state._timespinner_has_doublejump_of_npc(world, player) or state._timespinner_has_forwarddash_doublejump(world, player) or state._timespinner_has_fastjump_on_npc(world, player)),
+        LocationData('Forest', 'Forest: Green platform secret',  1337092, lambda state: state._timespinner_can_break_walls(world, player)),
+        LocationData('Forest', 'Forest: Rats guarded chest',  1337093),
+        LocationData('Forest', 'Forest: Waterfall chest 1',  1337094, lambda state: state.has('Water Mask', player)),
+        LocationData('Forest', 'Forest: Waterfall chest 2',  1337095, lambda state: state.has('Water Mask', player)),
+        LocationData('Forest', 'Forest: Secret Batcave',  1337096),
```

Comment:
This isn't a secret spot. This is the chest to the right of the Refugee camp that has 3 Plantbats and a Baby Cheveur in it

Reply:
ah right i was thinking it was the secret off to the right but thats the green platform secret. will fix